yaksh/templates/yaksh/question.html

```diff
@@ -198,10 +210,15 @@
          {% elif question.type == "upload" %}
          <br><button class="btn btn-primary" type="submit" name="check" id="check" onClick="return validate();">Upload</button>&nbsp;&nbsp;
          {% else %}
-         <button class="btn btn-primary" type="submit" name="check" id="check" onClick="submitCode();">Check Answer <span class="glyphicon glyphicon-cog"></span></button>&nbsp;&nbsp;
+           {% if notification == None %}
```

Comment:
Check answer  button will not be displayed if there is a notification.
So in case I submit a blank code, a notification will be generated and 
check answer button will disappear.